test/feature/README.md

```diff
@@ -31,3 +31,25 @@ In case you are using Rancher Desktop
 > Error: failed to setup compose cluster: Cannot connect to the Docker daemon at unix:///var/run/docker. sock. Is the docker daemon running?
 
 You need enable setting "Allow to acquire administrative credentials (sudo access)" in Preferences -> Application -> General
+
+
```

Comment:
Copy this text, paste it into the https://translate.yandex.ru/editor, and click on "Fix mistakes."